features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Shared/LeftMenuItem.jsx

```diff
@@ -64,23 +66,43 @@ const styles = theme => ({
     },
 });
 function LeftMenuItem(props) {
-    const { classes, theme, Icon } = props;
+    const [selected, setSelected] = useState(false);
+
+    const {
+        classes, theme, Icon, to, history, text,
+    } = props;
     const { leftMenu } = theme.custom;
     const strokeColor = theme.palette.getContrastText(theme.palette.background.leftMenu);
     const iconSize = theme.custom.leftMenuIconSize;
+    const ditectCurrentMenu = (location) => {
```

Comment:
Typo?

Reply:
Yes .. It should be "detect". Will fix this with next PR of the same component which I am currently working on.